lib/utils/visUtil.hpp

```diff
@@ -583,9 +584,13 @@ class StatTracker {
     /**
      * @brief Create a ring buffer.
      *
+     * @param name Buffer name.
```

Comment:
```suggestion
     * @param name The statistic's name.
```
Buffer gets used a lot to reference a very specific thing in the pipeline context, best to use another term when possible.